packages/modelviewer.dev/data/docs.json

```diff
@@ -67,6 +67,18 @@
           "default": "auto",
           "options": "auto, interaction, manual"
         }
+      },
+      {
+        "name": "with-credentials",
+        "htmlName": "withCredentials",
+        "description": "This attribute is a boolean value that indicates whether or not 3D model fetch should be made using credentials such as cookies, authorization headers or TLS client certificates.",
```

Comment:
Can you link to the MDN article? Also, if you can give a little lay-person detail on what situation you'd need this for, that would be great. For instance, we could simply make it always true; presumably there's a reason that's not a good idea?

Reply:
No problem, I'm updating the description.

> For instance, we could simply make it always true; presumably there's a reason that's not a good idea?

For backward compatibility and to follow default behaviour of `XMLHttpRequest.withCredentials` and three.js loader option, I kept them to `false` by default. Only a `true` value can change the value of three.js loader.

Regarding the always `true`, I imagine is to avoid sending authentication data if the request didn't ask for that and follow the principe : `privacy by design` ?

Reply:
> Regarding the always `true`, I imagine is to avoid sending authentication data if the request didn't ask for that and follow the principe : `privacy by design` ?

Yeah, that sounds right to me. Let's make sure our users understand that.

Reply:
I changed the text to that :

> This attribute make the browser include credentials (cookies, authorization headers or TLS client certificates) in the request to fetch the 3D model. It's useful if the 3D model file is stored on another server that require authentication. By default the file will be fetch without credentials. Note that this has no effect if you are loading files locally or from the same domain.